### PR TITLE
set volumeWhenMuted during GoogleIMAVideoWrapper#setVolume

### PR DESCRIPTION
**Pull request by dallen42**

### CODE REVIEW

**Comment by aeng7** [comment]
@dallen42 can we add a unit test for the use case this is solving? Thanks!

**Comment by dallen42** [comment]
@aeng7 added a unit test

**Comment by aeng7** [comment]
LGTM

File: js/google_ima.js [hunk 1]
```diff
@@ -2598,6 +2598,7 @@ require("../html5-common/js/utils/utils.js");
      */
     this.setVolume = function(volume, muteState)
     {
+      volumeWhenMuted = volume;
```
Comment (pilievOoyala) [inline js/google_ima.js]: shouldn't this be saved when we are actually about to mute and not everytime we set the volume?

Comment (aeng7) [reply js/google_ima.js]: The bug was because we were doing just that. We could rename the variable to savedVolume to avoid confusion.

Comment (dallen42) [reply js/google_ima.js]: ..except there is already a savedVolume in the GoogleIMA class. Might still be confusing. Will try and come up with something suitable

Comment (aeng7) [reply js/google_ima.js]: Shoot. Wonder if we can consolidate them in a future release.

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**